Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/LobbyViewModel.cs

```diff
@@ -81,12 +81,17 @@ public class LobbyViewModel : BaseViewModel {
             }
 
             webSocketService.StartGame += StartGame;
+            webSocketService.PlayerJoined += AddUser;
         }
 
         private async Task LoadUsers() {
             Users = await UnitOfWork.Instance.UserRepository.GetAll(GameModel.Id);
         }
 
+        private void AddUser(PlayerEventData data) {
+            Users.Add(data.Player);
+        }
```

Comment:
Ik denk dat je of van `Users` een `ObservableCollection` moet maken, of na de Add `OnPropertyChanged("Users")` moet aanroepen. ObservableCollection zou dan netter zijn.

Reply:
Het is niet nodig, want de Users wordt niet weergeven in de View. Er zijn al twee `ObservableCollection`'s, namelijk `Police` en `Thiefs`. Deze worden automatisch aangepast, wanneer de `Users` zijn aangepast.

Reply:
Goed punt, maar als Users.Add aangeroepen wordt, wordt de OnPropertyChanged voor Thiefs en Police ook niet aangeroepen. Met een ObservableCollection gebeurt dat denk ik wel. 
Maar zoals ik in mijn andere review zei moet er vgm nog een API aanpassing gedaan worden voordat dit getest kan worden.